ipn/ipnlocal/local.go

```diff
@@ -1737,6 +1722,8 @@ func (b *LocalBackend) Start(opts ipn.Options) error {
 			b.logf("failed to save UpdatePrefs state: %v", err)
 		}
 		b.setAtomicValuesFromPrefsLocked(pv)
+	} else {
+		b.setAtomicValuesFromPrefsLocked(b.pm.CurrentPrefs())
```

Comment:
Note that this is just what migrateStateLocked always did as a side effect before, when it was nil. So I moved it here, because in the UpdatePrefs != nil path above it's already called.

It might be pointless here, but it's what happened before.

Reply:
seems safe